blockchain/vm/interpreter.go

```diff
@@ -82,13 +75,15 @@ type Interpreter struct {
 	returnData []byte // Last CALL's return data for subsequent reuse
 }
 
-// NewInterpreter returns a new instance of the Interpreter.
-func NewInterpreter(evm *EVM, cfg *Config) *Interpreter {
+// NewEVMInterpreter returns a new instance of the Interpreter.
+func NewEVMInterpreter(evm *EVM, cfg *Config) *Interpreter {
 	// We use the STOP instruction whether to see
 	// the jump table was initialised. If it was not
 	// we'll set the default jump table.
 	if !cfg.JumpTable[STOP].valid {
-		cfg.JumpTable = ConstantinopleInstructionSet
+		var jt JumpTable
+		jt = ConstantinopleInstructionSet
+		cfg.JumpTable = jt
```

Comment:
Why do we need this change instead of direct assignment like before?

Reply:
thanks, i'll remove it.